package.json

```diff
@@ -55,6 +55,8 @@
     "react-dom": "^15.3.0 || ^16.0.0-beta.1"
   },
   "dependencies": {
+    "@types/enzyme": "^2.8.4",
```

Comment:
The first issue is that it's preventing the documentation to build. The second is that it's increasing the installation duration. Styled-components put them as devDependencies.